docs/applications/simple-bank-account/README.md

```diff
@@ -128,12 +132,13 @@ scalar.dl.client.private_key_path=conf/client-key.pem
 If everything is set up properly you should be able to register your certificate on the ScalarDL network as
 
 ```bash
-$ ${SCALAR_SDK_HOME}/client/bin/scalardl register-cert --properties ./conf/client.properties
+$ cd contract
+$ ${SCALAR_SDK_HOME}/client/bin/scalardl register-cert --properties ./contract/conf/client.properties
 ```
 
 You should receive status code 200 if successful.
 
-To register your contracts you can create a `contracts.toml` file in the `conf` directory using the following format:
+To register your contracts you can create a `contracts.toml` file in the `contract/conf` directory using the following format:
```

Comment:
Ditto. I couldn't see the directory `contract/conf`.

Reply:
Thank you! I fix it in e894a75.